browser/src/layer/tile/CommentSection.ts

```diff
@@ -274,7 +274,7 @@ export class Comment extends CanvasSectionObject {
 		imgAuthor.setAttribute('width', this.sectionProperties.imgSize[0]);
 		imgAuthor.setAttribute('height', this.sectionProperties.imgSize[1]);
 
-		if (this.sectionProperties.docLayer._docType === 'text') {
+		if (this.sectionProperties.docLayer._docType !== 'spreadsheet') {
```

Comment:
This worries me somehow. Impress and Draw don't have reply counts and this seems structurally wrong. Also there may be some other code which checks reply count div (didn't check though).

Reply:
it is intended because even in Draw and Impress comments get collapsed so it is helpful to indicate when a comment is collapsed while it is being edited. On the other hand we do not collapse comments in calc